README.md

```diff
@@ -19,3 +20,6 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+The runtime complexity of the brute-force algorithm, which involves generating all permutations of the input list, is $O(n!)$ in the worst case as there are $n!$ possible permutations. For each permutation, the algorithm involves swapping elements in the array (a constant time operation) and checking if the permutation is sorted, which takes $O(n)$ time. Therefore, the overall time complexity for my brute-sort algorithm, for checking each permutation and sorting checks, is $O(n! * n)$ in the worst case. The best-case complexity is the first permutation tried, in the already sorted list resulting in a best-case time complexity of $O(1)$ . If we randomly created permutations without memory, I believe the worst-case time complexity would remain $O(n!)$, but the specific number of attempted permutations before finding the sorted list could vary. However it wouldn’t change the overall complexity. 
```

Comment:
...but you're always generating all permutations, right? So even if the first one is the sorted array, the *overall* complexity wouldn't be constant (and it can't be, as we discussed in class).